### PR TITLE
Auto-adds several infowindow CSS classes

### PR DESCRIPTION
**Pull request by javierarce**
This PR removes the need to add certain CSS classes. Instead, it uses the required .js- classes from the template to add them.

### CODE REVIEW

**Comment by javierarce** [comment]
cc: @xavijam, can you take a look, please?

File: src/geo/ui/infowindow.js [hunk 1]
```diff
@@ -178,8 +181,34 @@ var Infowindow = View.extend({
...
+    }
+    if (hasTitle) {
+      $infowindow.addClass('has-title');
+    }
+  },
+
   _renderScroll: function () {
-    if (this.$('.has-scroll').length === 0) return;
+    if (this.$('.js-content').length === 0) {
+      return;
```
Comment (xavijam) [inline src/geo/ui/infowindow.js]: Shouldn't we remove has-scroll in case it is added?

**Comment by xavijam** [comment]
For me it makes sense, so :+1:

File: src/geo/ui/infowindow.js (hunk 1 above)
Comment (javierarce) [inline src/geo/ui/infowindow.js]: Mmm… why would someone add it?

Comment (xavijam) [inline src/geo/ui/infowindow.js]: Not really common, just wondering :dancer:

Comment (javierarce) [inline src/geo/ui/infowindow.js]: :ok_hand:  :D

**Pull request merged**